guides/common/modules/snip_host-registration-steps.adoc

```diff
@@ -32,13 +32,30 @@ ifndef::katello,satellite,orcharhino[]
 ----
 # hammer host-registration generate-command
 ----
++
+If your hosts do not trust the SSL certificate of {ProjectServer}, add the `--insecure` flag to the registration command.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# hammer host-registration generate-command
+--insecure 1
```

Comment:
should this be `--insecure true`? I guess both work as far as the API is concerned, but using a true (lol) boolean seems more correct here?